### PR TITLE
[Linux] Fix undefined variables at rescue or always block when test_setup failed

### PR DESCRIPTION
**Pull request by keirazhang**
If test case fails at test_setup, some variables used in rescue or always block may be not defined but used, which would lead to variable undefined error. This fix is to add conditions to use those variables.

### CODE REVIEW

**Review by linhuali — DISMISSED** [review]
LGTM. Thanks!

File: linux/check_inbox_driver/check_inbox_driver.yml [hunk 1]
```diff
@@ -59,31 +59,36 @@
...
           include_tasks: get_inbox_drivers.yml
           when: guest_os_ansible_distribution != "FreeBSD"
-        
+
         - name: "Get FreeBSD inbox drivers"
           include_tasks: get_freebsd_inbox_drivers.yml
           when: guest_os_ansible_distribution == "FreeBSD"
 
       rescue:
         - include_tasks: ../../common/test_rescue.yml
```
Comment (123lzxm) [inline linux/check_inbox_driver/check_inbox_driver.yml:68]: Add name?

Comment (keirazhang) [reply linux/check_inbox_driver/check_inbox_driver.yml]: Done. Thanks!

**Review by 123lzxm — APPROVED** [review]
LGTM, thanks.

**Pull request merged**